Vic2ToHoI4/Source/V2World/Pops/Pop.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Pop.h"
+#include "../Issues/Issues.h"
+
+
+
+float Vic2::Pop::getIssueSupport(const std::string& issueName) const
+{
+	if (const auto& issue = popIssues.find(issueName); issue != popIssues.end())
+	{
+		return issue->second;
+	}
+	else
```

Comment:
Redundant else.

Reply:
I prefer it to be there, for the reasons highlighted in this post: https://www.fluentcpp.com/2018/08/24/how-to-design-early-returns-in-c-based-on-procedural-programming/